src/components/publicApi/PublicLoginScreen.tsx

```diff
@@ -89,9 +92,26 @@ export function LoginScreen(props: Props): JSX.Element {
         username: props.username,
         customPermissionsFunction: props.customPermissionsFunction
       }}
-      initialAction={initializeLogin(theme, branding)}
+      initialAction={
+        props.initialRoute
+          ? initialRouteNameToAction(props.initialRoute)
+          : initializeLogin(theme, branding)
```

Comment:
The `initializeLogin` action is absolutely essential, since it sets up a bunch of security-related stuff. We cannot skip it. It also decides which scene to route to, so we just need to tweak that logic.

Maybe `initializeLogin` could take a third parameter, which gives the initial scene choice. If it's `undefined`, we auto-choose. If it's `'new-account'`, we go to the create flow, and if it's `'login'`, we either do password or PIN login as appropriate.